v3-nginx/docker-entrypoint.d/91-modsecurity-envsubst.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+# vim:sw=2:ts=2:sts=2:et
+
+set -eu
+
+LC_ALL=C
+ME=$( basename "$0" )
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+touch /etc/modsecurity.d/modsecurity-override.conf 2>/dev/null || { echo >&2 "$ME: error: can not modify /etc/modsecurity.d/modsecurity-override.conf (read-only file system?)"; exit 0; }
```

Comment:
```suggestion
touch /etc/modsecurity.d/modsecurity-override.conf 2>/dev/null || { echo >&2 "$ME: ERROR: can not modify /etc/modsecurity.d/modsecurity-override.conf (read-only file system?)"; exit 0; }
```

Reply:
This is the same as upstream, so I was keeping it compatible. If you want I can change it...

Reply:
This came from upstream, but removed in new iteration.

Reply:
Nope, fine now since it's consistent in the script.